internal/api/handlers/adapters/schedulers.go

```diff
@@ -0,0 +1,169 @@
+package adapters
```

Comment:
@arthur29 What do you think about renaming this package to `request_adapters` or something like this? We already have an `adapters` package for out ports adapters structure.

Reply:
Done